pgjdbc/src/test/java/org/postgresql/test/jdbc4/jdbc41/GetObjectTest.java

```diff
@@ -694,6 +694,39 @@ void getBlob() throws SQLException {
     }
   }
 
+  /**
+   * Test the behavior getObject for bytea columns.
+   */
+  @Test
+  void getBytea() throws SQLException {
+    Statement stmt = conn.createStatement();
```

Comment:
WDYT of moving the declaration of `stmt` closer to its usage?

Reply:
Ok